site/content/docs/5.1/components/scrollspy.md

```diff
@@ -14,20 +14,26 @@ Refer to [Orange navbar]({{< docsref "/components/orange-navbar" >}}) and [Nav &
 
 ## How it works
 
-Scrollspy toggles the `.active` class on anchor (`<a>`) elements when the element with the `id` referenced by the anchor's `href` is scrolled into view. Generally, it will be most useful in conjunction with a Boosted [nav component]({{< docsref "/components/navs-tabs" >}}) or [list group]({{< docsref "/components/list-group" >}}), but it will also work with any anchor elements in the current page.
+Scrollspy toggles the `.active` class on anchor (`<a>`) elements when the element with the `id` referenced by the anchor's `href` is scrolled into view. Scrollspy is best used in conjunction with a Boosted [nav component]({{< docsref "/components/navs-tabs" >}}) or [list group]({{< docsref "/components/list-group" >}}), but it will also work with any anchor elements in the current page. Here's how it works.
```

Comment:
With the actual changes, I wonder if we could remove the alert from ODS about this component since it could use branded components to work with.

Reply:
Tried to move the callout for the specific Navbar part via https://github.com/Orange-OpenSource/Orange-Boosted-Bootstrap/pull/1247/commits/24b0ecdd6570db6b8f98eb19de13c5955817bca5. Better like this?